resources/dicts/events/death/general.json

```
@@ -1785,7 +1785,7 @@
         "sub_type": ["murder"],
         "tags": ["all_lives"],
         "weight": 20,
-        "event_text": "m_c agreed to venture with r_c out onto a frozen pond. m_c went first while r_c hung back. m_c slid around playfully, taunting the other cat to chase {PRONOUN/m_c/object}. {PRONOUN/m_c/subject/CAP} {VERB/m_c/were/was} about to launch another teasing remark when the ice cracked below {PRONOUN/m_c/poss} paws. r_c lingered to make sure {PRONOUN/m_c/subject} didn't resurface before running back to camp with a yowl.",
+        "event_text": "r_c tricked m_c into playing on thin ice. {PRONOUN/r_c/subject/CAP} watched m_c disappear beneath the surface.",
```

{PRONOUN/r_c/subject/CAP} watched m_c disappear beneath the surface, all trace of play wiped from their calculating expression.",

---

I feel like that lengthens it a bit too much, but I recognize that my version is a bit vague.  Perhaps something like 
```suggestion
        "event_text": "r_c tricked m_c into playing on thin ice. {PRONOUN/r_c/subject/CAP} watched carefully as m_c disappear beneath the surface.",
```
would work? It implies more intent to ensure m_c goes down.

---

Love it, perf!